RoboJackets/robocup-software
# Distraction fix

## Description
A fix for distraction

## Steps to test
### Test Case 1
1. Set up game to be in attack goal situation
2. Run game with distraction
Expected result:  Passes shouldn't be too close and the pass locations shouldn't be the same every time.
